packages/lib/modules/swap/SwapProvider.tsx

```diff
@@ -87,9 +85,9 @@ function selectSwapHandler(
     return new AuraBalSwapHandler(tokens)
   }
 
-  if (pool && poolActionableTokens) {
-    return new PoolSwapHandler(pool, poolActionableTokens)
-  }
+  // if (pool && poolActionableTokens) {
```

Comment:
Once you delete this, it is possible that you can remove pool and poolActionableTokens params from `selectSwapHandler` function.